src/fetch_purchases_suppliers.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import pandas as pd
+
+
+BASE_DATA_DIR = './data/'
+
+def fetch_suppliers():
+    raw_suppliers = []
+    first_response = requests.get('http://compras.dados.gov.br/fornecedores/v1/fornecedores.json').json()
+    raw_suppliers = raw_suppliers + first_response['_embedded']['fornecedores']
+
+    last_page = first_response['count']
+    for page in range(2, last_page):
+        print('{}/{}'.format(page, last_page))
+        response = requests.get('http://compras.dados.gov.br/fornecedores/v1/fornecedores.json').json()
+        raw_suppliers = raw_suppliers + first_response['_embedded']['fornecedores']
+    return raw_suppliers
+
+
+def prepare_data(raw_data):
+    suppliers_attributes_columns = {'id': 'id', 'cnpj': 'cnpj', 'nome': 'name', 'ativo': 'active',
```

Comment:
This is quite difficult to read, what do you think about formatting it [like that](https://github.com/datasciencebr/serenata-de-amor/blob/master/src/clean_cnpj_info_dataset.py#L38-L59)?